Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library(['github.com/cloudogu/ces-build-lib@1.44.2', 'github.com/cloudogu/dogu-build-lib@v1.3.0', 'github.com/cloudogu/zalenium-build-lib@v2.1.1'])
+@Library(['github.com/cloudogu/ces-build-lib@v1.48.0', 'github.com/cloudogu/dogu-build-lib@v1.4.1', 'github.com/cloudogu/zalenium-build-lib@v2.1.1'])
```

Comment:
As far as I know we use already Cypress tests for our E2E Tests. Maybe we should remove the unused zalenium build lib.

Reply:
Fixed it myself